templates/invoices_app/invoice-list.html

```diff
@@ -28,18 +33,36 @@ <h1 class="text-center"> {% trans "My Invoices"%}</h1>
                 </tr>
             </thead>
             <tbody>
-            {% if not object_list%}
-            <tr><td  class="table-danger text-center" colspan="5">{% trans "No invoices" %} <a href="{% url 'invoice-create' taxpayer_id=taxpayer_id %}">Create one?</a></td> </tr>
+            {% if not is_AP %}
+              {% if not object_list%}
+              <tr><td  class="table-danger text-center" colspan="5">{% trans "No invoices" %} <a href="{% url 'invoice-create' taxpayer_id=taxpayer_id %}">Create one?</a></td> </tr>
+              {% endif %}
             {% endif %}
             {% for invoice in object_list %}
                 <tr>
                     <th scope="row">{{invoice.id}}</th>
                     <td>{{invoice.invoice_date_received}}</td>
-                    <td><span class="badge badge-success">{{invoice.status}}</span></td>
+                    <td>
+                      <span>
+                        <strong>Status:</strong>
+                        {% if invoice.status == 'NEW' or  invoice.status == 'PAID' or invoice.status == 'APPROVED' %}
```

Comment:
Here you could use the constants defined for the different status. Check this comment https://github.com/evbeda/suppliers_management/pull/81#discussion_r330220817